libdvdcss2.mk

```diff
@@ -0,0 +1,47 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS        += libdvdcss2
+LIBDVDCSS2_VERSION := 1.4.2
+DEB_LIBDVDCSS2_V   ?= $(LIBDVDCSS2_VERSION)
+
+libdvdcss2-setup: setup
+	wget -q -nc -O$(BUILD_SOURCE)/libdvdcss2-$(LIBDVDCSS2_VERSION).tar.bz2  https://download.videolan.org/pub/libdvdcss/$(LIBDVDCSS2_VERSION)/libdvdcss-$(LIBDVDCSS2_VERSION).tar.bz2
```

Comment:
use `wget -P $(BUILD_SOURCE)` since you arent change the filename

Reply:
I am changing the filename. libdvdcss-1.4.2.tar.bz2 becomes libdvdcss**2**-1.4.2.tar.bz2